server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -389,8 +390,10 @@ public MappingLookup mappingLookup() {
      * Returns all mapped field types.
      */
     public Iterable<MappedFieldType> getEagerGlobalOrdinalsFields() {
-        return this.mapper == null ? Collections.emptySet() :
-            this.mapper.mappers().fieldTypesLookup().filter(MappedFieldType::eagerGlobalOrdinals);
+        if (this.mapper == null) {
+            return Collections.emptySet();
+        }
```

Comment:
I'm afraid to ask, but do we have tests for this method? :)

Reply:
Do you know, I don't think we do.  I'll add some!

Reply:
thank you, it makes sense maybe that we don't have it because we were testing the other method, but now that that method is gone, it makes sense to test this one? :)